packages/usa-footer/src/usa-footer.twig

```diff
@@ -1,45 +1,34 @@
 <footer class="usa-footer {% if modifier %} {{ modifier }} {% endif %}">
-  <div class="grid-container usa-footer__return-to-top">
+  <div class="usa-footer__container usa-footer__return-to-top">
     <a href="#">{{ footer.return_to_top | default('Return to top') }}</a>
   </div>
   <div class="usa-footer__primary-section">
-    {% block primary %}
-      {% include "@components/usa-footer/src/_includes/usa-footer-nav-primary.twig" with footer.primary_section.nav %}
-    {% endblock %}
+    <div class="grid-container">
+      {% block primary %}
+        {% include "@components/usa-footer/src/_includes/usa-footer-nav-primary.twig" with footer.primary_section.nav %}
+      {% endblock %}
+    </div>
   </div>
   <div class="usa-footer__secondary-section">
-    <div class="grid-container">
+    <div class="usa-footer__container">
       {% block secondary %}
         <div class="grid-row grid-gap">
-          <div class="usa-footer__logo grid-row mobile-lg:grid-col-6 mobile-lg:grid-gap-2">
-            <div class="mobile-lg:grid-col-auto">
+          <div class="grid-col-12 mobile-lg:grid-col-6">
```

Comment:
A few concerns here:

1. The logo markup looks quite different from previous.
2. `grid-col-12` can be removed.